src/content.js

```diff
@@ -372,6 +372,26 @@ function addDiffViewWithoutWhitespaceOption() {
 	);
 }
 
+// When navigating with next/previous in review mode, maintain whitespace option.
+function addWhitespaceOptionToNextPrevious() {
+
+	// Only proceed if in review.
+	if (!select.exists('.pr-review-tools > .diffbar-item')) {
+		return;
+	}
+
+	const url = new URL(location.href);
+	const hidingWhitespace = url.searchParams.get('w') === '1';
+
+	for (const a of select.all('.commit > .BtnGroup.float-right > a.BtnGroup-item')) {
```

Comment:
The `commit` class is present on the commit header `<div>` when in PR review mode, though it's not clear to me what the purpose is of the other classes which are also present (specifically, `full-commit` and `prh-commit`). They might be more suitable or worth adding to this selector as well, and I'm happy to adjust.